server/src/serializer/ReviewSerializer.js

```diff
@@ -9,6 +9,13 @@ class ReviewSerializer {
 
     const relatedUser = await review.$relatedQuery("user")
     serializedReview.username = relatedUser.username
+
+    const relatedReviews = await review.$relatedQuery("votes")
+    let votesSum = 0
+    for(review of relatedReviews){
+      votesSum += review.value
+    }
```

Comment:
this is fine, but would encourage you to create a VoteSerializer so that this Vote specific logic doesn't live inside the Review Serializer (and means that we can total the votes in other places if needed)